cosmwasm/packages/sgx-vm/src/instance.rs

```diff
@@ -87,7 +87,9 @@ where
         let module = compile(code)?;
         Instance::from_module(&module, deps, gas_limit)
         */
-        let enclave = get_enclave().map_err(EnclaveError::sdk_err)?;
+        let enclave = get_enclave()
+            .map_err(EnclaveError::sdk_err)?
+            .ok_or(EnclaveError::EnclaveBusy {})?;
         let module = Module::<S, Q>::new(
```

Comment:
This is the context where queries try to access the enclave. If it's busy (`Ok(None)` was returned) then we tell the users about it.